NicolaeChristof/Rise
# Enhancement leaffalling ui animation

This adds an animation for leaves falling off the UI branch when the tree player grows a branch. Additionally, I removed a lot of the unused UI references in UIController and Settings Controller.

How to Test:
Collect acorns and plant leaves to make sure the animation is triggering correctly. Also checking to see that it's functioning correctly across the seasons.
